ts/src/currencycom.ts

```diff
@@ -1361,6 +1365,34 @@ export default class currencycom extends Exchange {
         return this.parseOrder (response, market);
     }
 
+    async fetchOrder (id, symbol = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchOrder() requires a symbol argument');
```

Comment:
we have some helper methods to make this check shorter `this.checkRequiredSymbol('fetchOrder', symbol);`